.github/workflows/master-mirrors-main.yaml

```diff
@@ -0,0 +1,19 @@
+name: 'master mirrors this branch'
```

Comment:
Either rename `this` to `main` or rename the whole thing to `sync master with main`.
The reason here is that when we look at the Github actions UI, we should be able to figure out what is the CI job.

Reply:
```suggestion
name: 'sync master with main'
```